src/lib/components/submit/SubmitSteps.svelte

```diff
@@ -1,12 +1,18 @@
 <script lang="ts">
 	import Step from '$lib/components/ui/Step.svelte';
+	import { derived } from 'svelte/store';
 	import { userInitialized, userNotSignedIn, userSignedIn } from '$lib/derived/user.derived';
 	import { isNullish, nonNullish } from '@dfinity/utils';
 	import { userStore } from '$lib/stores/user.store';
 	import Aside from '$lib/components/ui/Aside.svelte';
+	import { governanceIdStore } from '$lib/derived/governance.derived';
+	import { GOVERNANCE_CANISTER_ID } from '$lib/constants/app.constants';
 
 	export let step: undefined | 'select' | 'write' | 'neuron' | 'review' | 'submitted' | 'readonly';
 
+	const showSelect = derived(governanceIdStore, ($governanceIdStore) => $governanceIdStore === GOVERNANCE_CANISTER_ID);
```

Comment:
this seems overcomplicated, does this works? why not just using an auto subscription to derive a state?

```
let something: boolean;
$: something = $storeSomething === 'something';
```